internal_ws/yktrace/src/tir.rs

```diff
@@ -472,6 +472,11 @@ impl VarRenamer {
         if let Some(v) = self.stack.last() {
             self.offset = *v;
         } else {
+            // presumably this fundamentally can't happen? at which point a `debug_assert` might be
+            // more appropriate so that the function is:
+            //   self.stack.pop();
+            //   debug_assert!(!self.stack.is_empty());
+            //   self.offset = self.stack.last().unwrap();
```

Comment:
It can be `self.offset = self.stack.last().expect("Unbalanced ...")`.

Reply:
I'd replace the `expect` with `unwrap`, but otherwise I agree!

Reply:
Fixed in 232c144.